data/json/vehicleparts/motor.json

```diff
@@ -43,14 +39,13 @@
     "item": "motor_small",
     "difficulty": 1,
     "durability": 120,
-    "power": 7460,
-    "energy_consumption": 8500,
+    "power": 7040,
```

Comment:
should be 7,500 W at 10,444 W consumption, making it the 7.5 kW generator running in reverse.

Reply:
72% efficiency is also quite low for an electric motor. While that would make sense for a generator, it really wouldn't for a motor.

Reply:
Same as alternator comment. Generators have lower efficiency than motors.